libs/transform.js

```diff
@@ -3,9 +3,10 @@
 const stats = require('simple-statistics')
 
 const util = require('./util')
-const {ExprBase} = require('./expr')
+const { ExprBase } = require('./expr')
 const DataFrame = require('./dataframe')
 const Summarize = require('./summarize')
+const { LOGIC_NULL } = require('blockly/msg/en')
```

Comment:
1. No spaces around `LOGIC_NULL`.
2. We aren't using Blockly messages anywhere else - delay that to a different PR?